### DISCUSSION TITLE
Anybody observe slow training speed for mamba compared to transformer model?

### DISCUSSION THREAD

**Discussion by vgthengane** [LMD0311/PointMamba#49] [General]

**Comment by d62lu** [comment]
yes, I just made the comparison. The speed of the Mamba block is truly slower than Transformer, under the same input dimension

**Comment by d62lu** [comment]
for both model training and inference. I am not looking into the details of the Mamba block, maybe I missed something in the code....

**Comment by LMD0311** [comment]
I've merged this issue into the “Discussion” section.

On my machine, with parameter expansion mamba has an advantage over Transformer. I'm not sure if this is environment/GPU related.
